### PR TITLE
Getting reports that UDH CSM messages with ucs2 are > 140 chars.

### PR DESCRIPTION
**Pull request by smn**
This issue investigates that report & tries to reproduce with a test.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

**Comment by smn** [comment]
:+1: when travis agrees and :wave: from Bangladesh.

File: vumi/transports/smpp/tests/test_mica.py [hunk 1]
```diff
@@ -34,9 +34,55 @@ def setUp(self):
...
+            " stone",
+            ", china or other materials, and may have a stem, handles or"
+            " other",
+            " adornments. Cups are used for drinking across a wide range of"
+            " cu",
+            "ltures and social classes, and different styles of cups may be"
+            " us",
+            "ed for different liquids or in different situations. Cups have"
+            " be",
+            "en used for thousands of years for the ...Reply 1 for more",
```
Comment (jerith) [inline vumi/transports/smpp/tests/test_mica.py]: Can we parenthesise the implicitly concatenated strings here? It's hard to tell which lines are supposed to be concatenated and which aren't, especially when the string content ends in ",".

**Comment by jerith** [comment]
:+1:, although I did leave a comment about readability that may be worth addressing.

**Comment by hodgestar** [comment]
Text reflowed and bracketed.

**Comment by jerith** [comment]
:+1:

**Pull request merged**